packages/easy-coding-standard/src/Console/Output/JsonOutputFormatter.php

```diff
@@ -36,8 +37,7 @@ public function report(ErrorAndDiffResult $errorAndDiffResult, Configuration $co
         $json = $this->createJsonContent($errorAndDiffResult);
         $this->easyCodingStandardStyle->writeln($json);
 
-        $errorCount = $errorAndDiffResult->getErrorCount();
-        return $errorCount === 0 ? Command::SUCCESS : Command::FAILURE;
+        return $this->retrieveReturnType->getReturnType($errorAndDiffResult, $configuration);
```

Comment:
This sounds like static analysis service.
Could you rename to service responsible for exit code?

E.g. `ExitCodeReporter->report()`

Reply:
Adjusted 👍

Reply:
The class name is the same.

Reply:
Forgot to rename a variable, fixed that one too.